app/services/process_submission_service.rb

```diff
@@ -134,19 +136,36 @@ def pdf_details(submission_details)
     submission_details.filter { |detail| detail.fetch('type') == 'pdf' }
   end
 
-  def generate_pdf(submission)
-    pdf_details(submission.submission_details).each do |pdf_detail|
-      GeneratePdf.new(pdf_generator_gateway: pdf_gateway(submission.service_slug)).execute(pdf_detail.with_indifferent_access)
-    end
+  def send_confirmation_email(pdf_detail, submission_id)
+    confirmation_email_service = SendConfirmationEmail.new(
+      email_service: EmailService,
```

Comment:
Unfortunate this EmailService is never instantiated.
Not difficult to change that but deferring that for now.